packages/yoastseo/src/languageProcessing/helpers/sentence/getSentences.js

```diff
@@ -53,6 +54,9 @@ export default function( text, memoizedTokenizer ) {
 	text = excludeEstimatedReadingTime( text );
 	// Unify only non-breaking spaces and not the other whitespaces since a whitespace could signify a sentence break or a new line.
 	text = unifyNonBreakingSpace( text );
+	// Remove images from text before tokenizing it into sentences.
+	// This step is done here so that applying highlight in captions is possible for all assessments that use this helper.
+	text = text.replace( new RegExp( imageRegex, "ig" ), "" );
```

Comment:
The step to remove images is done before splitting the text into sentences.
Adding this step here in `getSentences()` makes it possible not to duplicate this step in all assessments that need images to be stripped. 
Analysis on words and sentences level use this helper.
I think it's also fairly safe to do it here since we only need the images for the images assessments (i.e. Images, Image Keyphrase, and Image Alt tag assessments). In all those three assessments, we take the text directly from `Paper.getText()`.